RecoParticleFlow/Configuration/python/RecoParticleFlow_cff.py

```diff
@@ -97,10 +97,10 @@
 
     e.toModify(particleFlowBlock, useNuclear = cms.bool(False))
 
-    e.toModify(pfNoPileUpIso, enable = cms.bool(False))
-    e.toModify(pfPileUpIso, enable = cms.bool(False))
-    e.toModify(pfNoPileUp, enable = cms.bool(False))
-    e.toModify(pfPileUp, enable = cms.bool(False))
+    e.toModify(pfNoPileUpIso, enable = False)
```

Comment:
Shouldn't the name of the parameter be `Enable` (with upper-case E) everywhere? Please check!

In any case, as such it is horrible to see, and definitely error prone: why not to profit of this PR to rename the parameter with lower-case initial in the (very few) cases where it is written with the upper-case E?

Reply:
agree, done the renaming, consistently to lower case